src/14_cal.py

```diff
@@ -21,4 +21,40 @@
 
 import sys
 import calendar
-from datetime import datetime
\ No newline at end of file
+from datetime import datetime
+
+user = ()
+
+
+def prompt():
+    global user
+    user = input(
+        "Please provide a month, or a month and year separated by a comma.").split(',')
```

Comment:
very nicely done here, i love how you broke out the prompt from the main loop. that's excellent programming architecture! you were also very descriptive on how to input the information.